src/core/components/Settings/components/NotificationSettings.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+import { Box, H3, Stack, Text } from '@noom/wax-component-library';
+
+import { SwitchSetting } from './SwitchSetting';
+
+import { NotificationSettings as Settings } from '../models';
+
+export type NotificationSettingsProps = {
+  title?: string;
+  spacing?: number;
+  isLoading?: boolean;
+  settings: Settings;
+  onChangeGlobal: (type: string, isChecked: boolean) => void;
+  onChangeCommunity: (communityId: string, isChecked: boolean) => void;
+};
+
+export function NotificationSettings({
+  title,
+  isLoading,
+  spacing = 4,
+  settings,
+  onChangeGlobal,
+  onChangeCommunity,
+}: NotificationSettingsProps) {
+  const { formatMessage } = useIntl();
+
+  const sortedCommunities = [...settings.community].sort((a, b) =>
+    a.communityName.localeCompare(b.communityName),
+  );
+
+  return (
+    <Box>
+      {title && <H3 pb={spacing}>{title}</H3>}
+      <Stack divider={<Box />} spacing={spacing}>
+        <SwitchSetting
```

Comment:
could we also use a for here? this way, if we're adding a new type of notif, this doesn't have to be changed

Reply:
Hmm, we can. Ill update.